lib/tasks/export.rake

```diff
@@ -1,24 +1,11 @@
-require 'csv'
-
 namespace :export do
-  def present_subscriber_list(list_id)
-    list = SubscriberList.find(list_id)
-    { subscriber_list_id: list_id, title: list.title, count: list.subscribers.count }
-  rescue ActiveRecord::RecordNotFound
-    warn "could not fetch record for #{list_id}"
-  end
-
-  desc "Export the number of subscribers for a collection of lists as a csv, accepts multiple arguments"
-  task :csv, [:subscriber_list_id] => :environment do |_, args|
-    CSV($stdout, headers: %i[subscriber_list_id title count], write_headers: true) do |csv|
-      rows = args.to_a.map { |list_id| present_subscriber_list(list_id) }
-      rows.compact.each { |row| csv << row }
-    end
+  desc "Export the number of subscriptions for a collection of lists as a csv, accepts multiple arguments"
```

Comment:
If I'm understanding this right could we change this to say "accepts multiple list IDs" or something?  Took me some double-checking to be sure what args to give.